### PR TITLE
Add Cypress tests for bulk upload

### PR DESCRIPTION
**Pull request by jaseemuddinn**
Related to #11002

### CODE REVIEW

File: site/cypress/e2e/Cypress-Gradeable/bulkupload.spec.js [hunk 1]
```diff
@@ -24,5 +24,50 @@ describe('Test cases revolving around bulk uploading', () => {
...
+
+        it(`${user} should be able to upload and split by QR code`, () => {
+            cy.login(user);
+            cy.visit(['sample', 'gradeable', 'bulk_upload_test']);
+            cy.get('#radio-bulk').click();
+            cy.get('#use-qr').check();
+            cy.get('#bulk-upload-file').selectFile('cypress/fixtures/bulk_upload_qr.pdf', { force: true });
+            cy.get('#bulk-upload-submit').click();
+            cy.get('.alert-success').should('contain', 'Bulk upload successful');
+        });
```
Comment (DarthNyan) [inline site/cypress/e2e/Cypress-Gradeable/bulkupload.spec.js:36]: Bulk uploading should trigger a daemon job and inform the user of the job's status. Can you add a check to verify this (for QR and page count)

File: site/cypress/e2e/Cypress-Gradeable/bulkupload.spec.js [hunk 2]
```diff
@@ -24,5 +24,50 @@ describe('Test cases revolving around bulk uploading', () => {
...
+
+        it(`${user} should be able to delete uploads`, () => {
+            cy.login(user);
+            cy.visit(['sample', 'gradeable', 'bulk_upload_test']);
+            cy.get('#radio-bulk').click();
+            cy.get('#bulk-upload-file').selectFile('cypress/fixtures/bulk_upload_qr.pdf', { force: true });
+            cy.get('#bulk-upload-submit').click();
+            cy.get('.alert-success').should('contain', 'Bulk upload successful');
+            cy.get('#bulk-upload-delete').click();
+            cy.get('.alert-success').should('contain', 'Bulk upload deleted');
```
Comment (DarthNyan) [inline site/cypress/e2e/Cypress-Gradeable/bulkupload.spec.js:56]: Similarly here, we should verify not only that the 'Bulk upload deleted' banner appears, but that which we deleted has actually been deleted.

**Review by DarthNyan — CHANGES_REQUESTED** [review]
This is a really good start. See the documentation on [PR titles](https://submitty.org/developer/getting_started/make_a_pull_request). Other than that, there are just a few additional checks I think should be added.

File: site/cypress/e2e/Cypress-Gradeable/bulkupload.spec.js [hunk 3]
```diff
@@ -24,5 +24,50 @@ describe('Test cases revolving around bulk uploading', () => {
...
+            cy.login(user);
+            cy.visit(['sample', 'gradeable', 'bulk_upload_test']);
+            cy.get('#radio-bulk').click();
+            cy.get('#use-qr').check();
+            cy.get('#bulk-upload-file').selectFile('cypress/fixtures/bulk_upload_qr.pdf', { force: true });
+            cy.get('#bulk-upload-submit').click();
+            cy.get('.alert-success').should('contain', 'Bulk upload successful');
+            cy.visit(['sample', 'gradeable', 'bulk_upload_test', 'grading', 'details']);
+            cy.get('[data-testid="view-sections"]').click();
+            cy.get('[data-testid="grade-button"]').first().click();
```
Comment (IDzyre) [inline site/cypress/e2e/Cypress-Gradeable/bulkupload.spec.js:69]: This is an example of a good selector. If you can add (if they aren't already there)  data-testids, and use them for the elements being selected in this test? You can edit the relevant twig files to add them. 

Good start!

**Review by IDzyre — CHANGES_REQUESTED** [review]

**Comment by jaseemuddinn** [comment]
I have made a new pull request with updated PR title, Do Check!
https://github.com/Submitty/Submitty/pull/11030

**Comment by IDzyre** [comment]
You can edit the title of a PR, so in the future, it would be better to do so to not lose prior reviews.

**Pull request closed without merging**